include/LVSS.hpp

```diff
@@ -1,13 +1,58 @@
-#pragma once
+#ifndef _LVSS_
+#define _LVSS_
+
+#include <EVT/dev/LCD.hpp>
+#include <EVT/io/CANopen.hpp>
+#include <EVT/io/GPIO.hpp>
+#include <EVT/io/SPI.hpp>
+#include <EVT/utils/log.hpp>
+#include <LVSS.hpp>
+#include <cstdio>
+#include <cstring>
+#include <dev/TPS2HB50BQ1.hpp>
+
+namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
 
 namespace LVSS {
 
+static constexpr uint8_t POWER_SWITCHES_SIZE = 3;
+
 /**
  * This is an example of a class for a board
  */
-class LVSS {
+class LVSS : public CANDevice {
 public:
+    static constexpr uint8_t NODE_ID = 42;
+
+    /**
+     * Placeholder constructor for the LVSS class
+     */
+    explicit LVSS(TPS2HB50BQ1 powerSwitches[POWER_SWITCHES_SIZE]);
```

Comment:
this is different than the actual constructor you currently have in your cpp.

Reply:
It's fine, gonna need to merge https://github.com/RIT-EVT/LVSS/pull/4 into it and then change the constructor again later.